### PR TITLE
Update translation_EN.json

### PR DESCRIPTION
**Pull request by discip**
minor changes

### CODE REVIEW

File: Translations/translation_EN.json [hunk 1]
```diff
@@ -213,21 +213,21 @@
...
-			"desc": "Desired max QC voltage the iron negotiates for"
+			"desc": "Desired max QC voltage, the iron negotiates for"
 		},
 		"PowerLimit": {
 			"text2": [
 				"Power",
 				"limit"
 			],
-			"desc": "Maximum power the iron can use <W=watt>"
+			"desc": "Maximum power, the iron can use <W=watt>"
```
Comment (alvinhochun) [inline Translations/translation_EN.json:230]: Unneeded comma.

File: Translations/translation_EN.json [hunk 2]
```diff
@@ -262,7 +262,7 @@
 				"Hall sensor",
 				"sensitivity"
 			],
-			"desc": "Sensitivity of the Hall effect sensor to detect sleep <O=off | L=low | M=medium | H=high>"
+			"desc": "Sensitivity of the Hall effect sensor, to detect sleep <O=off | L=low | M=medium | H=high>"
```
Comment (alvinhochun) [inline Translations/translation_EN.json:265]: I would consider changing the second part to "used to trigger sleep mode".

File: Translations/translation_EN.json [hunk 3]
```diff
@@ -297,14 +297,14 @@
 				"Power pulse",
 				"delay"
 			],
-			"desc": "Delay before keep-awake pulse is triggered (x 2.5s)"
+			"desc": "Delay, before keep-awake pulse is triggered <x 2.5s>"
```
Comment (alvinhochun) [inline Translations/translation_EN.json:300]: Unneeded comma.

File: Translations/translation_EN.json [hunk 4]
```diff
@@ -297,14 +297,14 @@
...
-			"desc": "Delay before keep-awake pulse is triggered (x 2.5s)"
+			"desc": "Delay, before keep-awake pulse is triggered <x 2.5s>"
 		},
 		"PowerPulseDuration": {
 			"text2": [
 				"Power pulse",
 				"duration"
 			],
-			"desc": "Keep-awake-pulse duration (x 250ms)"
+			"desc": "Keep-awake-pulse duration <x 0.25s>"
```
Comment (alvinhochun) [inline Translations/translation_EN.json:307]: I don't know... is using seconds really better than using milliseconds here?

**Review by alvinhochun — COMMENTED** [review]
Added some inline comments. They are just what I feel when reading the changes. I can't say my English is good enough to claim any of my comments to be definitely right.

File: Translations/translation_EN.json [hunk 5]
```diff
@@ -213,21 +213,21 @@
 				"Scrolling",
 				"speed"
 			],
-			"desc": "Speed info text scrolls past at <S=slow | F=fast>"
+			"desc": "Speed info, text scrolls past at <S=slow | F=fast>"
```
[the remark this answers is not in the document]
Comment (discip) [reply Translations/translation_EN.json:216]: You are totally right!
I'm sorry!
My bad!

**Pull request closed without merging**